labellab-client/src/components/navbar/css/navbar.css

```diff
@@ -36,6 +37,16 @@
   color: #000000;
 }
 
+.project-nav-title {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  padding: 0 0 0 1em;
+  font-family: Lato, 'Helvetica Neue', Arial, Helvetica, sans-serif;
+  font-size: 21px;
```

Comment:
If font-size will in rem, it will be better for responsiveness.

Reply:
@Tushar19varshney I've converted `px` to `rem`. Could you please review the latest commit?